### DISCUSSION TITLE
[Off] - O que você anda jogando por aí?

### DISCUSSION THREAD

**Discussion by waldemirflj** [frontendbr/forum#1255] [Discussão]
Meu último jogo completado foi Diablo 3/Pc e estou pensando em dedicar um tempo para um outro game. O que vcs andam jogando por ai? Alguém aí curte um MMO/RPG?

**Comment by hmilena** [comment]
Spider-Man e God of War! Acho que termino Spider primeiro \o/ Próximos: Shadow of the Tomb Raider e Red Dead Redemption 2.

**Comment by luizbills** [comment]
Ultimamente não estou jogando mais. Anos atrás gastei várias horas com awesomenauts. Mas o unico jogo que nunca me decepcionou foi Dungeon Crawl Stone Soup (um roguelike bem roots).

**Comment by waldemirflj** [comment]
Comecei a jogar  [Path of Exile](https://store.steampowered.com/app/238960/Path_of_Exile/) pela Steam. O jogo parece bem promissor e com ótimos  comentários. Caso alguém esteja jogando algum game da party da um alô ai.

**Comment by dtfialho** [comment]
Jogo World of Warcraft, mas tenho um PS4 também, o último que zerei foi o God of War.

**Comment by klarkc** [comment]
xbox, n jogo no pc ha um tempo rs. Meus jogos favoritos são Forza, Cities Skylines, Fallout 4. Estou esperando os preços dos games caírem com a queda do dolar pra comprar o Red 2

**Comment by felipefialho** [comment]
- Football Manager (desde sempre)
- Voltei a jogar Cities Skylines também...

**Comment by viniciusmeneses** [comment]
Comecei a jogar [Runescape](https://www.runescape.com/community) faz uns dias (antes jogava a versão antiga no começo do ano [Old School Runescape](https://oldschool.runescape.com/)), mas é um pouco deprimente jogar sozinho qualquer MMORPG. Também tava procurando esses dias algum outro MMORPG pra jogar ou qualquer outro jogo multiplayer.

**Comment by thismarcoantonio** [comment]
No momento MP tenho jogado fortnite e off The Witcher 3 e AC Brotherhood (relembrando velhos tempos)

**Comment by klarkc** [comment]
Po podíamos montar uma guilda dos devs aqui pra algum MMO haha

**Comment by viniciusmeneses** [comment]
Curti muito a ideia do @klarkc!

**Comment by waldemirflj** [comment]
Gostei da ideia @klarkc. Qual seria o jogo?

**Comment by klarkc** [comment]
N sei, aceito sugestões, faz um bom tempo q n jogo um nem sei como tá o ramo de MMO's. Ouvi sobre o Fallout 76, mas n sei se vai ser MMO nem se a galera curtiria esse genero. Outro q ouço bastante falar é o Black Desert, mas n sei se está ainda em beta

**Comment by viniciusmeneses** [comment]
Vou colocar aqui uns jogos que são grátis e multiplayer que eu lembro de sugestão **(nem todos são MMORPG)**:

- Maplestory 2
- Ragnarok
- Runescape
- Battlerite
- Dota 2
- Fortnite
- Guild Wars 2
- Blade & Soul
- Warframe
- Paladins
- BRAIN / OUT
- Warface
- Path of Exile
- Realm of the Mad God
- Team Fortress 2

Devem ter outros que eu já joguei, mas acabei esquecendo.

**Comment by klarkc** [comment]
Tenho conta no GW2, recomendo jogaço :D WvW é muito fera. Mas n tem muito sandbox (tipo archeage q tbm curti pacas)

**Comment by viniciusmeneses** [comment]
Já tentei jogar GW2, mas parei porque não consegui entender muito bem como funcionava o jogo, as quests e tals... Mas falam que é um MMORPG muito bom mesmo.

**Comment by ViniciusGularte** [comment]
Ultimo jogo que joguei foi civilizations5, de MMO ainda to esperando o lançamento do proximo elder scrols :smile:

**Comment by tunnes** [comment]
Eitcha ninguém ai curte passar raiva com um "lozin"? 👀 hahaha

**Comment by viniciusmeneses** [comment]
@tunnes Hahaha, curti muito por uns 4 anos seguidos, depois parei de jogar no começo desse ano pra focar e terminar o TCC, depois nem voltei mais a jogar. Pelo menos pra mim, nem compensa mais jogar... cada vez a comunidade fica pior.

**Comment by hhendrikk** [comment]
Dota 2

**Comment by tunnes** [comment]
@viniciusmeneses Boa! Também sou de 2014 haha, dei uma parada por conta da correria do final da facul e do trabalho, voltei recentemente mas só de vez em quando mesmo, a comunidade ta ficando meio zoada e 40min de partida é muito tempo pra um passatempo =S

**Comment by rodrigets** [comment]
Overwatch!

**Comment by dantas-dev** [comment]
A espera de fallout 76. Minha série favorita.

Enquanto isso fico no fortnite kkk

**Comment by didiraja** [comment]
Mano, to jogando **Assassin's Creed: Origins** e me divertindo pra cacete. É o primeiro AC realmente depois do _Black Flag_.

Além do **Overwatch**, companheiro de sempre.
